File: swift/Subscribe.swift
// Subscribe to real-time data example
import BinaryOptionsToolsUni

func subscribe() async throws {
    let client = try await PocketOption(ssid: "your-session-id")
    try await Task.sleep(nanoseconds: 5_000_000_000)
    
    let subscription = try await client.subscribe(asset: "EURUSD_otc", durationSecs: 60)
    print("Listening for real-time candles...")
    print("Subscription created successfully!")
}

Task { try await subscribe() }

File: ruby/subscribe.rb
# Subscribe to real-time data example
require_relative 'binary_options_tools_uni'

client = BinaryOptionsToolsUni::PocketOption.new("your-session-id")
sleep 5

subscription = client.subscribe("EURUSD_otc", 60)
puts "Listening for real-time candles..."
puts "Subscription created successfully!"
